resources/providers/awslib/aws.go

```diff
@@ -32,3 +36,11 @@ type AwsResource interface {
 	GetResourceName() string
 	GetResourceType() string
 }
+
+func GetRegion(region *string) string {
```

Comment:
IMO, we can add default client logic to the `getClient` func and use it across all providers.